doc/bootstrap.rst

```diff
@@ -6,22 +6,36 @@ A new Ceph cluster is deployed by bootstrapping a cluster on a single
 node, and then adding additional nodes and daemons via the CLI or GUI
 dashboard.
 
+The following example installs a basic three-node cluster. Each
+node will be identified by its prompt. For example, "[monitor 1]"
+identifies the first monitor, "[monitor 2]" identifies the second
+monitor, and "[monitor 3]" identifies the third monitor. This
+information is provided in order to make clear which commands
+should be issued on which systems.
+
+"[any node]" identifies any Ceph node, and in the context
+of this installation guide means that the associated command
+can be run on any node.
+
 Get ceph-daemon
 ===============
 
 The ``ceph-daemon`` utility is used to bootstrap a new Ceph Cluster.
-You can get the utility by either installing a package provided by
-your Linux distribution::
 
-  sudo apt install -y ceph-daemon   # or,
-  sudo dnf install -y ceph-daemon   # or,
-  sudo yum install -y ceph-daemon
+Use curl to fetch the standalone script::
+
+  [monitor 1] # curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/ceph-daemon/ceph-daemon
+  [monitor 1] # chmod +x ceph-daemon
+  [monitor 1] # sudo install -m 0755 ceph-daemon /usr/sbin    # optional!
```

Comment:
let's drop this line?

Reply:
Do you want to drop the "sudo install -m 0755 ceph-daemon /usr/sbin # optional" line?

Reply:
yeah

Reply:
I have dropped this line.